histomicsui/web_client/vue/components/EditHeatmapOrGridData.vue

```diff
@@ -0,0 +1,229 @@
+                if (this.interpretation === 'contour' || this.interpretation === 'chloropleth') {
+                    propsToSave['stepped'] = this.stepped;
+                }
+            }
+            this.$emit('submit', propsToSave);
+        },
+        cancelDialog() {
+            this.$emit('cancel');
+        }
+    },
+    mounted() {
+        if (this.colorRange) {
+            _.forEach(this.colorRange, (color, index) => {
+                this.colorRangeData.push({
+                    colorString: color,
+                    value: this.rangeValues[index]
+                });
+            });
+        }
+        if (this.type === 'griddata' && !this.minColor && !this.maxColor) {
+            if (this.colorRange.length > 0) {
+                this.minColor = this.colorRange[0];
+                this.maxColor = this.colorRange[this.colorRange.length - 1];
+            } else {
+                this.minColor = 'rgba(0, 0, 0, 0)';
+                this.maxColor = 'rgba(255, 0, 0, 1)';
+            }
+        }
+        this.updateRangeDataKeys();
+    }
+}
+</script>
+
+<template>
+    <div class="modal-dialog" v-if="this.colorRangeData">
+        <div class="modal-content">
+            <form class="modal-form">
+                <div class="modal-header">
+                    <button type="button" class="close"  @click="cancelDialog()" ref="close">
+                        <span>&times;</span>
+                    </button>
+                    <h4>{{ headerMessage }}</h4>
+                </div>
+                <div class="modal-body">
+                    <div class="errors" v-if="validationErrors.length > 0">
+                        <p>Errors</p>
+                        <ul>
+                            <li v-for="error in this.validationErrors" :key="error">{{ error }}</li>
+                        </ul>
+                    </div>
+                    <div class="form-group" v-if="this.type === 'heatmap'">
+                        <label for="h-griddata-radius">Radius</label>
+                        <input id="h-griddata-radius" class="input-sm form-control" type="number" min="1" v-model="radius">
+                    </div>
+                    <div class="form-group" v-if="this.colorRangeData.length > 0">
+                        <label for="h-griddata-range">Range Colors</label>
+                        <table id="h-griddata-range" class="table table-bordered table-condensed">
+                            <thead>
+                                <tr>
+                                    <th>Value</th>
+                                    <th>Color</th>
+                                    <th></th>
+                                </tr>
+                            </thead>
+                            <tbody>
+                            <tr v-if="this.type === 'griddata'">
+                                <td>
+                                    Min. color
```

Comment:
Let's spell this and Max. Color out: "Minimum Color" and have a tool tip that says "The color of values below the lowest range value".

Reply:
See 303d397